packages/core/src/components/Card/Content.tsx

```diff
@@ -98,6 +104,14 @@ function CardContent({
         src={afterImageSrc}
       />
     );
+
+    if (onAfterImageClick) {
+      afterContent = (
+        <button name="AfterContent" className={cx(styles.side_button)} onClick={onAfterImageClick}>
```

Comment:
curious why the `name` attributes?

Reply:
`type="button"` instead?

Reply:
heartbeat

Reply:
I don't agree with coupling internal technologies to an open source project.

Reply:
agreed

Reply:
Then please make sure in the card rewrite we have the ability to add whatever content we want to the side, because we are going to want to track the clicks on this.